### ISSUE TITLE
add shapes and icons to legend

### ISSUE THREAD

**Issue by efh0888** [rstudio/leaflet#191]
Is it currently possible to add shapes and/or icons to a legend?

**Comment by yihui** [comment]
I don't think it is currently possible. Legends are for colors only at the moment. That said, perhaps you can prepare the HTML code by yourself and use `addControl()` to add it to the map.

**Comment by efh0888** [comment]
Thanks for the reply. I might try that.

**Comment by jpvanek** [comment]
Has this issue been addressed in the last 6 years?
